src/react-components/input/ToggleInput.js

```diff
@@ -4,10 +4,15 @@ import classNames from "classnames";
 import styles from "./ToggleInput.scss";
 
 export const ToggleInput = memo(
-  forwardRef(({ className, label, value, disabled, ...rest }, ref) => {
+  forwardRef(({ className, label, description, value, disabled, ...rest }, ref) => {
     return (
       <label className={classNames(styles.toggleInput, { [styles.disabled]: disabled }, className)}>
-        {label && <span className={styles.label}>{label}</span>}
+        {label && (
```

Comment:
May be nice to vertically align the toggle inputs. Right now it looks a bit awkward to have each at its own distance from the right hand side:
![image](https://user-images.githubusercontent.com/4072106/98053686-00562680-1dee-11eb-8348-a5ebad6d36de.png)

Reply:
I flipped the toggle input layout so that the buttons are all aligned and the text is now to the right.